tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -484,6 +489,16 @@ def test_pdb_check_simultaneous_node_drains(
         time.sleep(30)
         try:
             drain_nodes([node_B])
+            # Check Mon failovers in 10mins, 15mins and 20 mins
+            timeout = [600, 300, 300]
+            for failover in timeout:
+                sample = TimeoutSampler(
+                    timeout=failover,
+                    sleep=10,
+                    func=helpers.check_mon_pods_eq_3,
+                )
+                if sample.wait_for_func_status(result=True):
+                    assert "There are more than 3 mons"
```

Comment:
This message is not accurate.
It should be "Number of mons != 3", and not necessarily "more than"